pkg/lightning/backend/backend.go

```diff
@@ -143,7 +146,7 @@ type AbstractBackend interface {
 
 	OpenEngine(ctx context.Context, config *EngineConfig, engineUUID uuid.UUID) error
 
-	CloseEngine(ctx context.Context, engineUUID uuid.UUID) error
+	CloseEngine(ctx context.Context, config *EngineConfig, engineUUID uuid.UUID) error
```

Comment:
can the table info be stored in `OpenEngine()`, so we don't need to modify `CloseEngine()`

Reply:
Not really. When lightning restart at import phase, only CloseEngine is called, in this situation, there is no tableInfo available